### DISCUSSION TITLE
无法添加阿里云、百度网盘等

### DISCUSSION THREAD

**Discussion by MilkTeaNo7** [AlistGo/alist#2289] [General]
用的dockers部署，版本为v3.3.0，只有如下部署方式

docker部署的命令如下
`docker run -d \
--name AList \
-d --restart=always \
-v /share/NAS-Download/AList:/opt/alist/data \
-p 5244:5244 \
xhofe/alist:latest`

![image](https://user-images.githubusercontent.com/74584122/201106076-b7ad376b-7070-4a5d-ae54-e4bd6b8431a0.png)

**Comment by MilkTeaNo7** [comment]
一键脚本装了也没有，V2的是有的

**Comment by Becods** [comment]
![image](https://user-images.githubusercontent.com/35525491/201271110-c4d0be3b-c748-4c92-a4e6-31beed7772f6.png)
你就不能往下滑吗

**Comment by MilkTeaNo7** [comment]
已经发现了，问题解决了，屏幕太宽的问题，没注意到最右侧的滚动条，谢谢了

蓝云Reyes ***@***.***> 于2022年11月11日周五 13:38写道：
